File: java/src/main/java/io/camunda/training/services/CustomerService.java
package io.camunda.training.services;

import org.springframework.stereotype.Component;

@Component
public class CustomerService {

  public double getCustomerCredit(String customerId) {
    return Double.valueOf(customerId.substring(customerId.length() - 2));
  }

  public double deductCredit(double customerCredit, double amount) {
    return customerCredit > amount ? 0.0 : amount - customerCredit;
  }
}

File: cSharp/Services/CustomerService.cs
namespace CamundaTraining.Services
{
    public class CustomerService
    {
        public double GetCustomerCredit(string customerId)
        {
            return double.Parse(customerId.Substring(customerId.Length - 2));
        }

        public double DeductCredit(double customerCredit, double amount)
        {
            return customerCredit > amount ? 0.0 : amount - customerCredit;
        }
    }
}